examples/.eslintrc.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
   rules: {
     'no-console': 0,
+    'import/no-extraneous-dependencies': 0,
+    'import/no-unresolved': 0,
```

Comment:
I think this should be moved into a specific eslint override only for the examples folder(and maybe tests). because this rule is nice to have to prevent accidentally importing devDependencies in production code

Reply:
This already is an override for the `examples` folder. The file is `examples/.eslintrc.js`.